src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Apprenticeship.cs

```diff
@@ -61,7 +61,7 @@ private void CheckIsStoppedForChangeOfParty()
             }
         }
 
-        private void CheckStartDateForChangeOfParty(DateTime startDate)
+        private void CheckStartDateForChangeOfParty(DateTime? startDate)
```

Comment:
this check will fail for a null start date

Reply:
done